index.html

```diff
@@ -594,52 +596,57 @@
 
 Note that the default value of **`subsurface_radius_scale`** is set at $(1, 0.5, 0.25)$ to approximate the effect of Rayleigh scattering. If we roughly approximate the wavelength corresponding to each of the RGB channels as $\lambda/\mathrm{nm} \approx 650, 550, 450$ and assume the radii scale like the reciprocal of the extinction, then since in Rayleigh scattering the extinction scales like $\lambda^{-4}$ for light of wavelength $\lambda$ the expected relative magnitudes of the radii are $(1,  (550/650)^4, (450/650)^4) \approx (1, 0.5, 0.25)$, hence the chosen default. This provides a slightly more realistic default look for the subsurface than resulting from gray radii.
 
 !!! Note: Future work
-   The particular equations for $S(\mathbf{A})$ and $\boldsymbol{\alpha}(\mathbf{A})$ presented here make certain approximations (e.g. assumptions about the dielectric interface IOR). Ideally the mapping would be generalized to an arbitrary dielectric interface.
-
    Since the scattered ray may emerge from a different point on the surface than the input point, the material will in general differ there. Thus on transmitting back out, it will encounter different layer properties. In principle this should be taken into account, but how to model subsurface scattering in the presence of varying subsurface properties is an open question.
 
 
 Subsurface params             | Label        | Type      | Range           | Norm       | Default              | Description
 ------------------------------|--------------|-----------|:---------------:|:----------:|:--------------------:|----------------------------------------------
 **`subsurface_weight`**       | Weight       | `float`   | $ [0, 1]      $ |            | $ 0                $ | Mix weight between subsurface and diffuse slabs
-**`subsurface_color`**        | Color        | `color3`  | $ [0, 1]      $ |            | $ (0.8, 0.8, 0.8)  $ | Subsurface color, i.e. the multi-scatter albedo of $V^\infty_\mathrm{subsurface}$
-**`subsurface_radius`**       | Radius       | `float`   | $ [0, \infty) $ | $ [0, 1] $ | $ 1                $ | Length scale of diffusion along the surface
-**`subsurface_radius_scale`** | Radius scale | `vector3` | $ [0, 1]^3    $ |            | $ (1.0, 0.5, 0.25) $ | RGB multiplier to **`subsurface_radius`**, giving the per-channel diffusion length
+**`subsurface_color`**        | Color        | `color3`  | $ [0, 1]      $ |            | $ (0.8, 0.8, 0.8)  $ | Subsurface color, controls the observed reflection color of $V^\infty_\mathrm{subsurface}$
+**`subsurface_radius`**       | Radius       | `float`   | $ [0, \infty) $ | $ [0, 1] $ | $ 1                $ | Length scale of MFP
```

Comment:
> Length scale of MFP

Using the word "scale" in the description of the parameter that doesn't include "scale" in the name seems a bit confusing. Maybe we could reword it. Here are some ideas:

"Length of MFP"
"Extinction MFP"
"Scale MFP"
"Base MFP"
"Average distance light travels along a ray, specifically the extinction MFP"

Reply:
This is maybe from habit, as "length scale" is a common phrase in physics (meaning the rough magnitude of the distance at which some effect is relevant).

Perhaps the term "scale" is confusing though since there is a parameter `subsurface_radius_scale` as well. It may make more sense to call the scalar radius the "scale", and the color just "radius" -- which is what we did in Standard Surface (except `subsurface_scale` was the name of the scalar length).

I do think it makes more sense for something called a "scale" to be a length scale (which in the physics terminology, is an actual length).